File: hellocharts/src/main/java/net/raquezha/lecho/hellocharts/animation/PieChartRotationAnimator.java
package net.raquezha.lecho.hellocharts.animation;

public interface PieChartRotationAnimator {

    int FAST_ANIMATION_DURATION = 200;

    void startAnimation(float startAngle, float angleToRotate);

    void cancelAnimation();

    @SuppressWarnings("unused")
    boolean isAnimationStarted();

    @SuppressWarnings("unused")
    void setChartAnimationListener(ChartAnimationListener animationListener);

}

File: hellocharts/src/main/java/lecho/lib/hellocharts/animation/PieChartRotationAnimator.kt
package lecho.lib.hellocharts.animation

interface PieChartRotationAnimator {
    fun startAnimation(startAngle: Float, angleToRotate: Float)
    fun cancelAnimation()
    val isAnimationStarted: Boolean
    fun setChartAnimationListener(animationListener: ChartAnimationListener?)

    companion object {
        const val FAST_ANIMATION_DURATION = 200
    }
}